name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/wizards/datatransfer/ReviewExtractedItemsPage.java

```diff
@@ -706,9 +708,15 @@ private void checkEntries(List<ExtractedEntry> entries)
         for (ExtractedEntry entry : entries)
         {
             entry.clearStatus();
-            for (ImportAction action : actions)
-                entry.addStatus(entry.getItem().apply(action, this));
+            for (ImportAction action : actions) {
+                ImportAction.Status actionStatus = entry.getItem().apply(action, this);
+                entry.addStatus(actionStatus);
+                if (actionStatus.getCode() == ImportAction.Status.Code.ERROR)
+                    errors.add(new IncorrectFileException(entry.getItem().getSubject().getNote() + ": " + actionStatus.getMessage())); //$NON-NLS-1$
```

Comment:
The "import actions" (which are checking the transactions) run not only after the initial import but also when the user switches the accounts. The reason is that if you switch to a USD account, the transaction now might be importable - or still not importable. Either way, the list of error messages should not be calculated dynamically.

I haved changed it now this way: save the extraction errors separately, merge the lists with the errors from checking the extracted items accounts the accounts.